jobs/grafana/templates/credentials

```diff
@@ -0,0 +1,4 @@
+[default]
```

Comment:
doh.  I'm a dill.  You are right.  
i went to take in your most recent changes before submitting the PR and managed to remove it from the spec file.  I'll update, rebase and send through.